### PR TITLE
RIA-3719:Review remote hearing in record hearing adjustment

### PR DESCRIPTION
**Pull request by divisathyan**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/RIA-3719

### Change description ###
TCW Review remote hearing in record hearing adjustment

**Does this PR introduce a breaking change?** (check one with "x")
```
[ ] Yes
[ x] No
```

### CODE REVIEW

File: e2e/features/record-agreed-hearing-requirements.feature [hunk 1]
```diff
@@ -60,6 +60,9 @@ Feature: Record agreed hearing requirements
     And I should see the text `Do not enter the reason the appellant made the request.`
     And I should see the text `The respondent will be able to see this information and must not be informed of the appellant's personal circumstances.`
     And I should see the text `Adjustment request`
+
+
+    And I record `false` for Remote hearing
```
Comment (bradsorour) [inline e2e/features/record-agreed-hearing-requirements.feature:65]: Possibly more descriptive to have the text as `Remote hearing requirement will not be reviewed`

File: e2e/features/record-agreed-hearing-requirements.feature [hunk 2]
```diff
@@ -134,6 +137,9 @@ Feature: Record agreed hearing requirements
     And I should see the text `Do not enter the reason the appellant made the request.`
     And I should see the text `The respondent will be able to see this information and must not be informed of the appellant's personal circumstances.`
     And I should see the text `Adjustment request`
+
+    And I record `true` for Remote hearing
```
Comment (bradsorour) [inline e2e/features/record-agreed-hearing-requirements.feature:141]: Possibly more descriptive to have the text as `Remote hearing requirement will be reviewed`

**Review by bradsorour — APPROVED** [review]
Approved with comment.

File: e2e/features/record-agreed-hearing-requirements.feature (hunk 1 above)
Comment (rohith-tiru) [reply e2e/features/record-agreed-hearing-requirements.feature:65]: Any reason why there need to be extra lines?

**Review by mkayad — APPROVED** [review]
LGTM

**Review by Lee365 — APPROVED** [review]
LGTM

**Pull request merged**